app/src/main/java/com/duckduckgo/app/bookmarks/service/SavedSitesImporter.kt

```diff
@@ -53,12 +54,18 @@ class RealSavedSitesImporter(
                 savedSitesParser.parseHtml(document, bookmarksRepository)
             }
 
-            savedSites.forEach {
-                when (it) {
-                    is SavedSite.Favorite -> favoritesRepository.insert(it)
-                    is SavedSite.Bookmark -> bookmarksDao.insert(BookmarkEntity(title = it.title, url = it.url, parentId = it.parentId))
-                }
+            savedSites.filterIsInstance<SavedSite.Bookmark>().map {
+                BookmarkEntity(title = it.title, url = it.url, parentId = (it as SavedSite.Bookmark).parentId)
+            }.apply {
```

Comment:
nit: I think `also {}` suits better here. Convention for `apply` is generally to mutate the receiver object and return it.

To be clear, not a blocking comment, just an FYI nit

Reply:
Thanks, updated! :+1: